notebooks/cloud_build_tfx_part_i.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    TFX_IMAGE_URI = "gcr.io/{gcp-project}/{name}"
</pre><p>may need to provide place holder for <code>gcp-project</code>  and <code>name</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/Model-Training-as-a-CI-CD-System/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Will add. Thanks

Reply:
I will merge this PR for now, we can add more later